smarts/env/hiway_env.py

```diff
@@ -173,7 +213,7 @@ def scenario_log(self):
             "mission_hash": str(hash(frozenset(scenario.missions.items()))),
         }
 
-    def seed(self, seed: int) -> int:
+    def seed(self, seed: int = None) -> int:
```

Comment:
Why is the default value `None` ? Seed value cannot be `None`.

Reply:
I added that since otherwise it was not following the  OpenAI gym's seed definition. But i will drop it for now.